### PR TITLE
docs(connect): upload gitbook markdown code

### PR DESCRIPTION
**Pull request by ClementIV**
## What is the purpose of the change

1.  upload gitbook markdown code

Close: #547 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by ClementIV** [comment]
@duhenglucky

File: docs/cn/GitBook/connector/replicator/rocketmq-replicator.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+```http
+http://${runtime-ip}:${runtime-port}/connectors/${rocketmq-replicator-name}
+ ?config={
+ "connector-class":"org.apache.rocketmq.replicator.RmqSourceReplicator",
+ "source-rocketmq":"xxxx:9876",
+ "target-rocketmq":"xxxxxxx:9876",
+ "replicator-store-topic":"replicatorTopic",
+ "taskDivideStrategy":"0",
+ "white-list":"TopicTest,TopicTest2",
+ target-cluster":"{RocketMQCluster-Target}",
```
Comment (gripson) [inline docs/cn/GitBook/connector/replicator/rocketmq-replicator.md:73]:
```suggestion
""target-cluster":"{RocketMQCluster-Target}",
```

File: docs/cn/GitBook/connector/replicator/rocketmq-replicator.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+```http
+http://localhost:8081/connectors/replicator-test/stop
+```
+
+{% hint style="info" %}
+更多参数设置说明与Connector操作请参考 
+
+* [replicator配置参数说明](replicator-can-shu-pei-zhi.md)
+* [RESTful 接口](../../rocketmq-connect-1/rocketmq-runtime/restful-jie-kou.md)
```
Comment (gripson) [inline docs/cn/GitBook/connector/replicator/rocketmq-replicator.md:133]: link is incorrect

File: docs/cn/GitBook/connector/replicator/rocketmq-replicator.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+* 停止示例
+
+```http
+http://localhost:8081/connectors/replicator-test/stop
+```
+
+{% hint style="info" %}
+更多参数设置说明与Connector操作请参考 
+
+* [replicator配置参数说明](replicator-can-shu-pei-zhi.md)
```
Comment (gripson) [inline docs/cn/GitBook/connector/replicator/rocketmq-replicator.md:132]: link is incorrect

File: docs/cn/GitBook/connector/replicator/rocketmq-replicator.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```bash
+$ cd target
+# 拷贝jar包到RocketMQ Runtime插件文件夹
+$ cp rocketmq-replicator-0.1.0-SNAPSHOT-jar-with-dependencies.jar /usr/local/connectors-plugin/
+```
+
+{% hint style="warning" %}
+**RocketMQ Runtime插件文件路径可配置，需要和Runtime参数中的插件路径配置相对应。**
+
+注：[Runtime 插件路径配置](../../rocketmq-connect-1/rocketmq-runtime/runtime-can-shu-pei-zhi.md#runtime-pei-zhi-can-shu-shuo-ming)
```
Comment (gripson) [inline docs/cn/GitBook/connector/replicator/rocketmq-replicator.md:53]: link is incorrect

**Comment by ClementIV** [comment]
@gripson thx ! 😊

**Pull request merged**